cfme/scripting/quickstart.py

```diff
@@ -1,13 +1,15 @@
 from __future__ import print_function
 
-import os
-import sys
 import argparse
-import subprocess
-import json
 import hashlib
+import json
+import subprocess
+import sys
 from pipes import quote
 
+import distro
```

Comment:
**REQUIRED** to actually use distro we have to vendor it, since its not there when quickstart runs,

i did away with that a while back in order to remove a dependency of the process